src/ncp/ncp_base_radio.cpp

```diff
@@ -51,20 +51,12 @@ namespace Ncp {
 // MARK: Raw Link-Layer Datapath Glue
 // ----------------------------------------------------------------------------
 
-void NcpBase::LinkRawReceiveDone(otInstance *, otRadioFrame *aFrame, otError aError)
-{
-    sNcpInstance->LinkRawReceiveDone(aFrame, aError);
-}
-
-void NcpBase::LinkRawReceiveDone(otRadioFrame *aFrame, otError aError)
+otError NcpBase::LinkRawPackRadioFrame(otRadioFrame *aFrame, otError aError)
```

Comment:
Suggest renaming to `PackRadioFrame`? 
I think the other methods with `LinkRaw` prefix have `LinkRaw` in the name since they are callbacks from `LinkRaw` module.